storeRabbitData.js

```diff
@@ -0,0 +1,212 @@
+const ExperimentModel = require('./models/ExperimentModel');
+const StatusModel = require('./models/StatusModel');
+const SensorModel = require('./models/SensorModel');
+
+
+// -------------------HELPERS--------------------- //
+
+
+// helper 1 : resutruring the data to made it suitable for the db
+const cleanData = (data) => {
+  // data to send in the database
+  if (!data.config) {
+    throw new Error('Error: wrong data');
+  }
+
+  const experimentData = {
+    timestamp: data.date.replace(/h/, '_').split('_'),
+    assimilationLog: data['log assim'],
+    neuralNetworkLog: data['log rn'],
+    parameters: data.config,
+    status: JSON.parse(data.statuts),
+    locationId: data.location ? 1 : 1,
+    sensorsPosition: JSON.parse(JSON.parse(data.geometrie)),
+  };
+
+  // change date fortmat
+  experimentData.timestamp = new Date(
+    experimentData.timestamp[0],
+    experimentData.timestamp[1],
+    experimentData.timestamp[2],
+    experimentData.timestamp[3],
+    experimentData.timestamp[4]
+  );
+
+  // show results
+  return experimentData;
+};
+
+// helper 2 : check of the experiment already exist in the db
+const checkDbForExperiment = async (experiment) => {
+  const checkExperimentOnDb = await ExperimentModel.experimentAlreadyExists(
+    experiment
+  );
+  return checkExperimentOnDb;
+};
+
+// helper 3 store expriment
+const exprimentStoring = async (dataToStore) => {
+  // check if the experiment already
+  const checkingExperiment = await checkDbForExperiment(dataToStore);
+
+  if (checkingExperiment) {
```

Comment:
Fait le check dans l'autre sens et ne fait pas le cas ou ça existe, c'est déjà le pas défaut d'une fonction de renvoyer undefined si une ne renvoie rien
```suggestion
  if (!experimentExists) {
```